docs/source/guide/events.rst

```diff
@@ -1,24 +1,24 @@
-Extensibility Guide
+Extensibility guide
 ===================
 
-All of Boto3's resource and client classes are generated at runtime.
+All of Boto 3's resource and client classes are generated at runtime.
 This means that you cannot directly inherit and then extend the
 functionality of these classes because they do not exist until the
 program actually starts running.
 
 
 However it is still possible to extend the functionality of classes through
-Boto3's event system.
```

Comment:
We should make sure we're consistent with this. The [official page](https://aws.amazon.com/sdk-for-python/) and our readme all say "Boto3" but some of the docs have "Boto 3". I think the former is what we actually want, but we should verify.

Reply:
Boto3 seems to be consistent with marketing/branding. I'll comb through the docs now and will post and updated PR. Thanks!